app/src/androidTest/java/com/github/sdp/tarjetakuna/ui/ScannerFragmentTest.kt

```diff
@@ -37,6 +33,7 @@ class ScannerFragmentTest {
     fun setUp() {
         Intents.init()
         scenario = launchFragmentInContainer()
+        Thread.sleep(100) // Wait for the fragment to be created (otherwise the test fails because there is no view)
```

Comment:
same here, wait for the view to be created with a scheduler or polling (doing a while loop of check + wait)